backend/lcfs/db/models/Category.py

```diff
@@ -0,0 +1,20 @@
+from sqlalchemy import Column, Integer, Sequence, String, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+
+from lcfs.db.base import BaseModel, Auditable, EffectiveDates
+
+class Category(BaseModel,Auditable, EffectiveDates):
+    __tablename__ = 'category'
+    __table_args__ = {'comment': "Transfer Category"}
+
+
+    id = Column(Integer, Sequence('issuance_id'), comment="Unique identifier for the transfer category", primary_key=True, autoincrement=True)
```

Comment:
need to update our id names to prefix with the table name. for example id becomes category_id

Reply:
changes updated